### PR TITLE
DAOS-6129 doc: Add newlines in vos README.md

### PR DESCRIPTION
**Pull request by jolivier23**

### CODE REVIEW

**Comment by jolivier23** [comment]
I'm not sure what some of the math characters were supposed to be as they are garbage in vim and in markdown.   

This change is mostly just separating lines to make it easier to comment on reviews

**Comment by jolivier23** [comment]
The ticket number is wrong.   Should be 2169 so whoever lands it should fix it.   It would be helpful if we could land this

File: src/vos/README.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
+The Versioning Object Store (VOS) is responsible for providing and maintaining a persistent object store that supports byte-granular access and versioning.
+It maintains its own metadata in persistent memory and may store data either in persistent memory or on block storage, depending on available storage and performance requirements.
```
Comment (gnailzenh) [inline src/vos/README.md]: I think storage selection is not part of VOS, so it's probably better to clarify it.

File: src/vos/README.md [hunk 2]
```diff
@@ -37,80 +41,152 @@ This document contains the following sections:
...
+This enables a disruptive change in performance compared to conventional storage systems for application and system metadata, and small, fragmented and misaligned I/O.
+Direct access to byte-addressable low-latency storage opens up new horizons where metadata can be scanned in less than a second without bothering with seek time and alignment.
+
+The VOS relies on a log-based architecture using persistent memory primarily to maintain internal persistent metadata indexes.
+The actual data can be stored either in persistent memory directly or in block-based storage (via SPDK for instance).
+The DAOS service has two tiers of storage: Storage Class Memory (SCM) for byte-granular application data and metadata, and NVMe for bulk application data.
+Similar to how PMDK is currently used to faciliate access to SCM, the Storage Performance Development Kit (SPDK) is used to provide seamless and efficient access to NVMe SSDs.
+The current DAOS storage model involves three DAOS server xstreams per core, along with one main DAOS server xstream per core mapped to an NVMe SSD device.
```
Comment (gnailzenh) [inline src/vos/README.md]: three server xstreams per target (VOS pools+SPKD blobs), not per core.

Comment (jolivier23) [reply src/vos/README.md]: I’m not changing text here.  Just separating lines to make future reviews easier.  The only change in this patch other than adding new lines between sentences is to remove all Unicode characters (or fix them).  If we could defer edits to another patch, that would be helpful

File: src/vos/README.md (hunk 1 above)
Comment (jolivier23) [reply src/vos/README.md]: Same comment as below.  I haven’t edited anything yet

File: src/vos/README.md [hunk 3]
```diff
@@ -37,80 +41,152 @@ This document contains the following sections:
...
 
-NVML  is an open source collection of libraries for using persistent memory, optimized specifically for NVRAM. NVML is actually a collection of six libraries among which the libpmemobj library implements relocatable persistent heaps called NVML pools. This includes memory allocation, transactions, and general facilities for persistent memory programming. Locks can be embedded with PM-resident data structures, which are reinitialized (i.e. unlocked) automatically every time the NVML pool is opened. This property makes sure that all locks are always released when the NVML pool is opened.
+NVML is an open source collection of libraries for using persistent memory, optimized specifically for NVRAM.
+NVML is actually a collection of six libraries among which the libpmemobj library implements relocatable persistent heaps called NVML pools.
+This includes memory allocation, transactions, and general facilities for persistent memory programming.
+Locks can be embedded with PM-resident data structures, which are reinitialized (i.e.
+unlocked) automatically every time the NVML pool is opened.
+This property makes sure that all locks are always released when the NVML pool is opened.
```
Comment (gnailzenh) [inline src/vos/README.md]: not sure we need to introduce NVNL lock at here, because there is no use-case in DAOS

**Pull request merged**